projects/valtimo/dossier/src/lib/components/dossier-detail/dossier-detail.component.html

```diff
@@ -18,6 +18,19 @@
   [ngClass]="{'main-content--no-horizontal-overflow': tabHorizontalOverflowDisabled()}"
 >
   <div class="container-fluid">
+    <div
+      class="mb-2"
```

Comment:
this is a bootstrap class which is not used anymore. Please style through stylesheet with correct spacing from figma design